website/docker/README.md

```diff
@@ -0,0 +1,68 @@
+# thingsboard.github.io
+
+Based on [ruby:2.7.7](https://hub.docker.com/_/ruby). For detailed info, see the origin repository.
+
+## Deployment of the site in docker
+
+The commands below set up the environment for running GitHub pages in the docker. You do not need to install additional dependencies and packages, everything is already built into the docker image.
+
+If you do not have docker installed, you need to install it. You can do this by following the installation instructions: [Docker Engine installation overview](https://docs.docker.com/engine/install/)
+
+
+### Deploy the site using the docker command
+
+Please replace the `THINGSBOARD_WEBSITE_DIR` with the full path to your existing thingsboard.github.io repository.
+
+```bash
+docker run --rm -d -p 4000:4000 --name thingsboard_website --volume="THINGSBOARD_WEBSITE_DIR:/website" thingsboard/website
+```
+If you don't already have a local repository, download it:
+
+```bash
+git clone https://github.com/thingsboard/thingsboard.github.io.git website
+```
+
+### Deploy the site using the docker-compose
+
+Put the site contents and the docker-compose file in the same directory:
+
+```bash
+mkdir tb_docs && cd $_
```

Comment:
The same as above - ask user first to clone the docs project first, and as a second step - create docker-compose.yml file nearby and start docs as docker compose.